src/wallet/test/wallet_tests.cpp

```diff
@@ -385,8 +385,8 @@ class CreateTransactionTestSetup : public TestChain100Setup
     const std::string strTooSmallToPayFee = "The transaction amount is too small to pay the fee";
     const std::string strTooSmallAfterFee = "The transaction amount is too small to send after the fee has been deducted";
     const std::string strTooSmall = "Transaction amount too small";
-    const std::string strUnableToLocateCoinJoin1 = "Unable to locate enough CoinJoin non-denominated funds for this transaction.";
-    const std::string strUnableToLocateCoinJoin2 = "Unable to locate enough CoinJoin denominated funds for this transaction. CoinJoin uses exact denominated amounts to send funds, you might simply need to mix some more coins.";
+    const std::string strUnableToLocateCoinJoin1 = "Unable to locate enough non-denominated funds for this transaction.";
+    const std::string strUnableToLocateCoinJoin2 = "Unable to locate enough denominated funds for this transaction. CoinJoin uses exact denominated amounts to send funds, you might simply need to mix some more coins.";
```

Comment:
```suggestion
    const std::string strUnableToLocateCoinJoin2 = "Unable to locate enough mixed funds for this transaction. CoinJoin uses exact denominated amounts to send funds, you might simply need to mix some more coins.";
```

Ops, here too then

Reply:
nah, this one makes sense imo - you do need _mixed funds_ in _denominated amounts_ ;)